pyface/ui/qt4/init.py

```diff
@@ -34,6 +34,17 @@
 _app = QtGui.QApplication.instance()
 
 if _app is None:
+    try:
```

Comment:
It might be good to leave a link to the underlying issue here - https://github.com/enthought/pyface/issues/581 - for future reference.